src/v2/providers/https.ts

```diff
@@ -41,28 +41,57 @@ export { Request, CallableRequest, FunctionsErrorCode, HttpsError };
  * Options that can be set on an individual HTTPS Cloud Function.
  */
 export interface HttpsOptions extends Omit<GlobalOptions, 'region'> {
-  /* HTTP functions can override and specify more than one regions. */
+  /** HTTP functions can override global options and can specify multiple regions to deploy to. */
   region?: SupportedRegion | string | Array<SupportedRegion | string>;
+  /** If true, allows CORS on requests to this function.
+   * If this is a String or ReqExp, allows requests from domains that match the provided string or RegExp.
```

Comment:
Should we backtick String and RegExp (as contrasted with the lowercase "the string" below?  

Suggest "If set to `String` or . . ."  and "If set to `Array`, ..."